exasol_integration_test_docker_environment/lib/base/luigi_log_config.py

```diff
@@ -0,0 +1,61 @@
+import contextlib
+import os
+import tempfile
+from pathlib import Path
+from typing import Optional
+
+import jinja2
+import logging
+from exasol_integration_test_docker_environment.lib import PACKAGE_NAME
+from exasol_integration_test_docker_environment.lib.config.build_config import build_config
+
+global_log_file: Optional[Path] = None
```

Comment:
Why is this global logfile "var" needed?

Reply:
Unfortunately the log-config for Luigi is global and can be configured only during the first invocation of luigi.build. Thus I need to safe the original log path, and later I use this variable to check if it has been changed. The variable is only being used for that check, in validate_log_file() (If there were static variables in Python, I would use that ;) ).